.github/workflows/pre-release.yml

```diff
@@ -29,7 +29,7 @@ jobs:
     strategy:
       matrix:
         os: ['ubuntu-latest']
-        python-version: [3.9, '3.10']
+        python-version: [3.9, '3.10', '3.11']
```

Comment:
Keep all versions of just a few, such as 3.9 and 3.11?

Reply:
This works. Compared to the CircleCI jobs, these take no time, so I'm fine risking over-coverage.